### DISCUSSION TITLE
网页内容提取

### DISCUSSION THREAD

**Discussion by camaker** [apache/hertzbeat#2759] [Ideas]
HTTP 监控里有JsonPath 解析，我想从HTML页面里提取 某些数据的话怎么做呢？ 支持 CSS选择器 或者 Xpath 之类的解析器不？

**Comment by tomsun28** [comment]
refer this pr https://github.com/apache/hertzbeat/pull/2593, still on working.

**Comment by camaker** [comment]
tks
